cmd/daemon/main.go

```diff
@@ -92,35 +94,41 @@ var (
 		Flags: []cli.Flag{
 			&cli.UintFlag{
 				Name:  flagRPCPort,
-				Usage: "port for the daemon RPC server to run on; default 5001",
+				Usage: "Port for the daemon RPC server to run on",
+				Value: defaultRPCPort,
 			},
 			&cli.UintFlag{
 				Name:  flagWSPort,
-				Usage: "port for the daemon RPC websockets server to run on; default 8080",
+				Usage: "Port for the daemon RPC websockets server to run on",
+				Value: defaultWSPort,
 			},
 			&cli.StringFlag{
-				Name:  flagBasepath,
-				Usage: "path to store swap artefacts",
+				Name:  flagDataDir,
+				Usage: "Path to store swap artifacts", //nolint:misspell
```

Comment:
I looked in a Canadian dictionary and it said that Canadians spell artifacts the same as Americans.  :) In a computer context, I suspect even people in the UK use the above spelling.